include/mbedtls/mbedtls_config.h

```diff
@@ -2800,6 +2800,18 @@
  */
 #define MBEDTLS_GCM_C
 
+/**
+ * \def MBEDTLS_GCM_LARGETABLE
+ *
+ * Enable the Galois/Counter Mode (GCM) using large tables.
+ *
+ * Module:  library/gcm.c
+ *
+ * Requires: MBEDTLS_GCM_C
+ *
+ */
+//#define MBEDTLS_GCM_LARGETABLE
```

Comment:
It might be better to use `MBEDTLS_AES_FEWER_TABLES` rather than introduce a new option.

Reply:
`MBEDTLS_AES_FEWER_TABLES` only relates to AES tabes, however, GCM can also be used with other block ciphers. Additionally, not only the ROM tables have different sizes in my case but also the GCM context will be larger as more information has to be stored per key. Hence, I think it is beneficial to have another define for configuring the GCM optimization.